internal/gatewayapi/translator.go

```diff
@@ -895,8 +895,8 @@ func (t *Translator) ProcessHTTPRoutes(httpRoutes []*v1beta1.HTTPRoute, gateways
 								}
 								if !canAddHeader {
 									parentRef.SetCondition(
-										v1beta1.RouteConditionResolvedRefs,
-										metav1.ConditionFalse,
+										v1beta1.RouteConditionAccepted,
+										metav1.ConditionTrue,
 										v1beta1.RouteReasonUnsupportedValue,
 										fmt.Sprintf("RequestHeaderModifier Filter already configures request header: %s to be added/set, ignoring second entry", headerKey),
 									)
```

Comment:
ditto, probably shouldn't be setting a condition here

Reply:
https://github.com/envoyproxy/gateway/pull/495#discussion_r992829787